.gitignore

```diff
@@ -9,3 +9,5 @@
 *.csv
 !open_data_info.csv
 aws-s3/parcel-tmp
+package*.json
```

Comment:
I figure if we start adding more Node dependencies to the docs site build process, it might eventually make sense to keep `package.json` and `package-lock.json` under version control. Until then, we keep it out for the sake of simplicity.